Sources/App/configure.swift

```diff
@@ -28,9 +28,11 @@ private let jiraProjects = [
     "PRO"   : 16980, // Professional Services
     "PRSCR" : 16840, // Prescriptions
     "REFER" : 17244, // Referrals (next-gen)
+    "RM"    : 17403,  // Real-Time Matching
```

Comment:
Why do they need to be whitelisted at all btw? What happens if the fix version gets created in a non-whitelisted project? Will show up on the CRP ticket and can be removed if not desired, no?

Seems like there's a lot of maintenance around this for not much gain.

As it says on the top the ID can be acquired through the API if it's needed.

Reply:
https://github.com/babylonhealth/babylon-ios/blob/develop/Documentation/Process/Release%20process/CRP-Bot-ImplementationDetails.md#creating-the-jira-release--updating-the-fix-version-fields-createandsetfixedversions

> This whitelist has 2 main purposes […]

Reply:
> Why do they need to be whitelisted at all btw?
So that the bot can create fix version on projects and their subsequent tickets. This is as per our sSDLC, so is a requirement to comply. All changes on the changelog have to be scraped into the CRP tickets.

> What happens if the fix version gets created in a non-whitelisted project?
It won't. Projects that are not whitelisted throw the exception `Project X is not part of our whitelist for creating JIRA versions`.

> Will show up on the CRP ticket and can be removed if not desired, no?
Correct, but that then requires a full manual review of the CRP ticket contents every time, removing the benefit of automating the capture and documenting of changes.

> Seems like there's a lot of maintenance around this for not much gain.
You don't see the gain because you're not working with it in that aspect, I am. The gain in making these little changes is significant and reduces the manual overhead I have.

Reply:
TL;DR:

 * not all boards are compatible with the bot. Some bots are "next-gen" JIRA boards that needs to be configured appropriately in JIRA first (especially to enable their Version fields in tickets) before they can work with the Bot
* We don't want to accidentally create JIRA Release and Fix Version on boards we don't own

Reply:
Thanks for the explanation folks! 🎩